pkg/readers/container_image_reader.go

```diff
@@ -0,0 +1,148 @@
+package readers
+
+import (
+	"context"
+	"fmt"
+	scalibr "github.com/google/osv-scalibr"
+	scalibrlayerimage "github.com/google/osv-scalibr/artifact/image/layerscanning/image"
+	"github.com/google/osv-scalibr/converter"
+	"github.com/google/osv-scalibr/extractor"
+
+	//"github.com/google/osv-scalibr/converter"
+	el "github.com/google/osv-scalibr/extractor/filesystem/list"
+	sl "github.com/google/osv-scalibr/extractor/standalone/list"
+	"github.com/safedep/vet/pkg/common/logger"
+	"github.com/safedep/vet/pkg/models"
+	"github.com/safedep/vet/pkg/parser"
+)
+
+type ContainerImageReaderConfig struct {
+	Image string
+}
+
+type containerImageReader struct {
+	config *ContainerImageReaderConfig
+}
+
+var _ PackageManifestReader = &containerImageReader{}
+
+func NewContainerImageReader(config *ContainerImageReaderConfig) (*containerImageReader, error) {
+	return &containerImageReader{
+		config: config,
+	}, nil
+}
+
+func (c containerImageReader) Name() string {
+	return "Container Image Reader"
+}
+
+func (c containerImageReader) ApplicationName() (string, error) {
+	return defaultApplicationName, nil
+}
+
+func (c containerImageReader) EnumManifests(handler func(*models.PackageManifest, PackageReader) error) error {
+	image, err := c.getScalibrContainerImage()
+	if err != nil {
+		logger.Errorf("failed to get Scalibr container image: %s", err)
+		return fmt.Errorf("failed to get Scalibr container image: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to get container image: %s", err)
```

`Scalibr` is internal library. It doesn't make sense to expose it as a error message to user. Logging is fine.